src/attributes.ts

```diff
@@ -625,6 +625,10 @@ const ATTRIBUTES_DEFINITION_BASE = {
 		name: 'Ankle_cuffs_chainable',
 		description: 'A cuff that is chainable to another item',
 	},
+	Handcover: {
```

Comment:
For consistency with the other cover attributes, it should be renamed to `Hand_cover`.

Reply:
Done.